.gitignore

```diff
@@ -4,4 +4,8 @@ __pycache__/
 __pycache__/
 
 3-4o_mini_fine_tuning/evals/*
-3-4o_mini_fine_tuning/data/*
\ No newline at end of file
+3-4o_mini_fine_tuning/data/*
+
+11-recommendation/images
+
+.venv/
```

Comment:
Are we actually using venv here? Should it be in the instructions readme?

Reply:
good call!